client/src/app/global-components/header/header.component.html

```diff
@@ -1,17 +1,17 @@
-<div class="navbar-wrapper">
-  <div class="navbar-wrapper_logo">
+<div class="header-wrapper">
+  <div class="header-wrapper_logo">
     <a href="/">Budgetify</a>
   </div>
-  <div class="navbar-wrapper_list">
+  <div class="header-wrapper_list">
     <ul>
       <li><a href="/categories">Categories</a></li>
```

Comment:
Don't use href for navigation. Use routerLink instead

Reply:
I use routerLink